Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -1185,15 +1204,15 @@ final class PledgeViewModelTests: TestCase {
         self.vm.inputs.stripeTokenCreated(token: "stripe_token", error: nil)
       )
 
-      self.goToThanks.assertDidNotEmitValue()
+      self.goToThanksProject.assertDidNotEmitValue()
 
       self.vm.inputs.paymentAuthorizationViewControllerDidFinish()
 
-      self.goToThanks.assertDidNotEmitValue()
+      self.goToThanksProject.assertDidNotEmitValue()
 
       self.scheduler.run()
 
-      self.goToThanks.assertValues([project])
+      self.goToThanksProject.assertValues([project])
```

Comment:
Should we assert the values of `goToThanksReward` and `goToThanksCheckoutData` here too? 🤔